docs/source/quickstart.mdx

```diff
@@ -1,296 +1,507 @@
-- [The Future of MCP is GraphQL](https://www.apollographql.com/blog/the-future-of-mcp-is-graphql)
+Create new tools by adding GraphQL operation files to your `operations/` directory:
 
-### Advanced Options
+```graphql
+# operations/GetProducts.graphql
 
-<details>
-<summary>Alternative ways to run the MCP Server</summary>
-
-#### Using STDIO Transport
-
-You can run the MCP Server using STDIO transport instead of Streamable HTTP. This is useful for certain environments or when you need more direct control over the server process.
-
-1. Download the binary of the latest version of Apollo MCP Server
-2. Use MCP Inspector to run the server:
-
-    ```yaml title="Config for stdio transport"
-    endpoint: https://thespacedevs-production.up.railway.app/
-    operations:
-      source: local
-      paths:
-      - <absolute-path-to-MCP-example-dir>/operations
-    schema:
-      source: local
-      path: <absolute-path-to-MCP-example-dir>/api.graphql
-    transport:
-      type: stdio
-    ```
+# Retrieves product information by ID
+query GetProducts($productId: ID!) {
+  product(id: $productId) {
+    id
+    name
+    price
+    description
+  }
+}
+```
 
-    ```sh
-    npx @modelcontextprotocol/inspector apollo-mcp-server <path to the preceding config>
-    ```
+Apollo MCP Server automatically detects new `.graphql` files and exposes them as tools. No server restart needed!
 
-3. Configure Claude Desktop to use STDIO:
+**Learn more:** [Defining MCP Tools](https://www.apollographql.com/docs/apollo-mcp-server/define-tools)
 
-    ```json
-    {
-      "mcpServers": {
-        "thespacedevs": {
-          "command": "<absolute-path-to-MCP-server-binary>",
-          "args": [
-            "<path to the preceding config>"
-          ]
-        }
-      }
+### Testing Operations in GraphOS Studio
+
+The `rover dev` command command includes Sandbox:
+
+1. Open http://localhost:4000 in your browser
+2. Test your operations directly:
+   - Browse your schema
+   - Run queries and mutations
+   - Experiment with variables
+   - View responses
+
+This is invaluable for:
+- Verifying operations before exposing them as MCP tools
+- Understanding your schema structure
+- Debugging query issues
+- Exploring available data
+
+## Step 5: Deploy Your MCP Server
+
+Once you're ready to deploy, Apollo MCP Server can run in any container environment.
+
+### Using the Apollo Runtime Container
+
+Your project includes a pre-configured `mcp.Dockerfile` for easy deployment. This container includes:
+
+- Apollo Router for serving your GraphQL API
+- Apollo MCP Server for MCP protocol support
+- All necessary dependencies
+
+**Build the container:**
+```bash
+docker build -f mcp.Dockerfile -t my-mcp-server .
+```
+
+**Run locally:**
+```bash
+docker run -p 4000:4000 -p 5050:5000 \
+  -e APOLLO_KEY=$APOLLO_KEY \
+  -e APOLLO_GRAPH_REF=$APOLLO_GRAPH_REF \
+  my-mcp-server
```

Comment:
I think we might need a MCP_ENABLE=1 since this is the AIO container? https://github.com/apollographql/apollo-runtime?tab=readme-ov-file#running-the-container